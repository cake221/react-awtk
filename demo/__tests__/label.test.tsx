import "../label"

describe("test button", ()=>{

  it("toMatchSnapshot", ()=>{
    expect(__AwtkSnapshot).toMatchSnapshot()
  })
})
