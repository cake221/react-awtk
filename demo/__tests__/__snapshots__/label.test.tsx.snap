// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test button toMatchSnapshot 1`] = `
Array [
  "getRootHostContext",
  "getChildHostContext",
  "shouldSetTextContent",
  "getChildHostContext",
  "shouldSetTextContent",
  "createInstance",
  "funtion  label_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "funtion  widget_set_self_layout_params( widget, x, y, w, h ) :undefined, center, middle, 50%, 30",
  "funtion  widget_use_style( widget, style ) :undefined, big_green",
  "funtion  widget_set_text_utf8( widget, text ) :undefined, ok",
  "finalizeInitialChildren",
  "createInstance",
  "funtion  window_create( parent, x, y, w, h ) :null, 0, 0, 0, 0",
  "appendInitialChild",
  "finalizeInitialChildren",
  "prepareForCommit",
  "appendNodeChild",
  "resetAfterCommit",
  "getPublicInstance",
  "funtion  widget_add_child( widget, child ) :undefined, [object Object]",
  "funtion  widget_layout( widget ) :undefined",
]
`;
